qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -509,7 +506,7 @@ def _execute_lua(self, _script: list, steps: int) -> np.ndarray:
         """
         nplc = self.nplc()
         linefreq = self.linefreq()
-        _time_trace_extra_visa_timeout = self._time_trace_extra_visa_timeout()
+        _time_trace_extra_visa_timeout = self._extra_visa_timeout
         estimated_measurement_duration = 2*1000*steps*nplc/linefreq
         new_visa_timeot = (estimated_measurement_duration
```

Comment:
Note that there's a small typo here, `new_visa_timeot` -> `new_visa_timeout`.

Reply:
ahh..thanks.